### PR TITLE
Creates ia-book-downloads component

### PR DESCRIPTION
**Pull request by shaneriley**
Provides a basic component to be slotted into the menu slider component for use in book navigator. Accepts an array of download items and an optional expiration days as a number.

book_downloads

### CODE REVIEW

File: src/icon_close.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { html } from 'lit-element';
+
+export default html`
+<svg height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><circle cx="9" cy="9" fill="#fff" r="9"/><path d="m10 6-3 3.08108108 3 2.91891892" stroke="#333" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/></g></svg>
+`;
```
Comment (jbuckner) [inline src/icon_close.js]: Do we already have this in `iaux-icons`?

**Review by jbuckner — APPROVED** [review]
One question, but looks good overall 👍

File: src/icon_close.js (hunk 1 above)
Comment (shaneriley) [reply src/icon_close.js]: No. This is what I mentioned in stand up, and what prompted my discussion with Jim about needing a lot of icons to be properly exported so I can make one update to ia-icons for all of these new components.

Comment (shaneriley) [reply src/icon_close.js]: This has now been removed and replaced with ia-icons

**Pull request merged**